README.md

```diff
@@ -29,6 +29,10 @@ A third party security audit was performed by Cure53, you can see the full repor
 
 `runc` only supports Linux. It must be built with Go version 1.19 or higher.
```

Comment:
In line 30, I have tested it and found that runc can’t compile in go 1.19 anymore, we should change it to ‘go 1.20 or higher’.

Reply:
Note: This is an error review if we’re doing review commit by commit.